app/Http/Backend/Controllers/ProductsController.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace App\Http\Backend\Controllers;
+
+use App\Models\Products;
+use Illuminate\Http\Request;
+
+use App\Http\Backend\Requests;
+use App\http\Backend\Requests\CreateProductsRequest;
+use App\http\Backend\Requests\UpdateProductsRequest;
+use App\Http\Backend\Controllers\Controller;
+use DB;
+use App\Models\Categories;
```

Comment:
I think model name should be single not plural. Could you please change from `Categories` to `Category`